docs/en/ingest-management/elastic-agent/debug-standalone-elastic-agent.asciidoc

```diff
@@ -204,112 +206,15 @@ Returns something like:
 [[inspect-configuration]]
 == Inspect the {agent} configuration
 
-To inspect the {agent} configuration, run the `inspect` command. For example:
+To inspect the running {agent} configuration use the <<elastic-agent-inspect-command>>.
```

Comment:
```suggestion
To inspect the running {agent} configuration use the <<elastic-agent-inspect-command,elastic-agent inspect>> command.
```
I notice that without providing text the link renders as "To inspect the running Elastic Agent configuration use the elastic-agent inspect." (i.e., without "command.")